src/vm/object.inl

```diff
@@ -389,7 +389,7 @@ inline void FindByRefPointerOffsetsInByRefLikeObject(PTR_MethodTable pMT, SIZE_T
             continue;
         }
 
-        SIZE_T fieldStartIndex = pFD->GetOffset() / sizeof(void *);
+        SIZE_T fieldStartIndex = pFD->GetOffset();
```

Comment:
Nit: This is offset not index. 

It can be just used directly on next line without being stored into a local field: 
`FindByRefPointerOffsetsInByRefLikeObject(pFieldMT, baseOffset + pFD->GetFieldOffset(), processPointerOffset);`